README.md

```diff
@@ -250,20 +267,22 @@ The **summary.tsv** output file generated by `staramr` contains the following co
 * __Genotype__: The AMR genotype of the isolate.
 * __Predicted Phenotype__: The predicted AMR phenotype (drug resistances) for the isolate.
 * __Plasmid__: Plasmid types that were found for the isolate.
+* __Scheme__: The organism that `MLST` has identified.
+* __Sequence Type__: The hashcode that's assigned when combining all allele types
```

Comment:
Instead of **hashcode** could you use the term **sequence type**.

Reply:
Fixed in my next commit